phing

```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env php
 <?php
 
-/**
- * Shell wrapper for Phing
- * $Id: 3da5a2758ca689a6ee49defabcda5efa6dd70a97 $
- */
-$phingPhpFilepath = __DIR__ . '/vendor/phing/phing/bin/phing.php';
+$phingPhpFilepath = __DIR__ . '/phing.phar';
```

Comment:
nitpick, the variable should be rather named `$phingPharFilepath` now

Reply:
I have renamed it